### DISCUSSION TITLE
Stuck: Unable to open grid in the tutorial "Non-Newtonian Blood Flow - FDA Nozzle"

### DISCUSSION THREAD

**Discussion by dtlinker** [nextfoam/baram#125] [General]
I have downloaded the grid file "jetmesh" and expanded it. When I try to open it in BARAMFlow I get "jetMesh is not a baram project." The folder has a folder "polymesh" and a file local.cfg. I have also tried opening polymesh, and each of the folders within it, and get the same result. What am I missing? Thanks

**Comment by kdg5424** [comment]
Hello @dtlinker. This is Donggyu from NEXTFOAM.
There was something wrong with the polyMesh. Sorry for inconvenience.
I've made the polyMesh with baramMesh again and will post it on BARAM page.
Of course, I attach the polyMesh here (your post.)

Nozzle Mesh Project (baramMesh) : [nozzleMesh.zip](https://github.com/user-attachments/files/17357801/nozzleMesh.zip)
Nozzle Project (baramFlow) : [nozzle.zip](https://github.com/user-attachments/files/17357803/nozzle.zip)

Best regards

Donggyu Kim

**Comment by dtlinker** [comment]
That works better, but when I go through the BARAMFlow instructions on Materials configure blood, the popup for viscosity it is Constant and it and the edit button are grayed out, so I cannot change them to Carreau. I ran it with those unchanged parameters, and the residuals did not not change. Suggestions?
![image](https://github.com/user-attachments/assets/831c8301-ed35-4a26-8cff-b5126e90d243)

**Comment by kdg5424** [comment]
Hello @dtlinker.
Could you let me know your version of BARAM?
You can check it here : Help - About
Non-newtonian model is included in the version 24.4.0.
So if your version is lower than 24.4.0, I suggest you to install BARAM 24.4.0 from BARAM page

Thanks

Best regards

Donggyu Kim

**Reply by dtlinker** [reply]
I upgraded, and am now using 24.4.0. I was able to change to Carreau and tried running again, but the residuals plot does not look right. I am attaching it. Also, what file do I chose if I click the Paraview menu item? 
![image](https://github.com/user-attachments/assets/f3aacc5a-8698-4fcb-9919-8372535b7bb4)

**Comment by kdg5424** [comment]
Hello @dtlinker.
Could you check that you changed material in "region0"?
You should make the material named blood and change it in "region0". You can change in Cell Zone Conditions.
![image](https://github.com/user-attachments/assets/e9ab66a1-0c03-4bd8-80e5-eb03846bd575)

The residuals would be like the below picture.
![image](https://github.com/user-attachments/assets/45956f51-43f8-47f4-ba67-41018567e32d)

Thanks

Best regards

Donggyu Kim

**Reply by dtlinker** [reply]
I checked and it was correct. I tried deleting the "air" material to make it unavailable, and the result was the same. I tried restarting from scratch and got the same result. I am attaching the project as a zip file so maybe you can look at it and see what I have done wrong. Thank you for all your help.
[nozzle.zip](https://github.com/user-attachments/files/17389708/nozzle.zip)

**Comment by kdg5424** [comment]
Hello @dtlinker. This is Donggyu from NEXTFOAM.
I saw your project and found out something wrong.
Your Under-Relaxation factor was wrong.
In tutorial manual, we use the default Under-relaxation factors : Pressure for 0.3 / Momentum for 0.7.
![image](https://github.com/user-attachments/assets/fc2fe2e9-2e0b-4798-8a5d-742df503c265)
And the meanings that Under-relaxation factors is 1e-6 is the residuals should not be changed very well.

You should change "convergence criteria" to 1e-6 for pressure and momentum not Under-relaxation factor.
With default Under-relaxation factor, your project  also runs very well like below picture
![image](https://github.com/user-attachments/assets/a9abfca4-ef2f-4659-b7c0-946621a86271)

Best Regards

Donggyu Kim

**Reply by dtlinker** [reply]
Thank you! That fixed it!

**Reply by kdg5424** [reply]
You're welcome!!
I'm very glad that you fixed it!!

Thanks for your interest in our program!

**Comment by dtlinker** [comment]
All resolved. Thank you so much for the help.
